config/ccg_bert.conf

```diff
@@ -0,0 +1,33 @@
+include "defaults.conf"	
+include 
+pretrain_tasks = ccg	
+target_tasks = ccg	
+project_dir = ccg_exps
+bert_model_name = bert-large-uncased	
+tokenizer = ${bert_model_name}	
+exp_dir = ${bert_model_name}	
+run_dir = ""	
+bert_fine_tune = 1	
+do_target_task_training = 0	
+
+batch_size = 16	
+max_seq_len = 510	
+reload_task = 1	
```

Comment:
Why override these in the config file?

Reply:
max_seq_len definitely needs to be overriden because we add CLS and SEP tags, the others are not necessary so I'll remove.